tests/testSvt.py

```diff
@@ -7,6 +7,7 @@
 sys.path.append(os.path.dirname(os.path.abspath(__file__)) + "/lib")
 
```

Comment:
I think you can **revert all changes to this file**. The added test is not needed and you have a bad merge causing dead tests being re-added.

Reply:
Ok done!